oxidation/libparsec/crates/protocol/schema/authenticated_cmds/pki_enrollment_info.json

```diff
@@ -0,0 +1,80 @@
+[
+    {
+        "label": "PkiEnrollmentInfo",
+        "major_versions": [
+            2,
+            3
+        ],
+        "req": {
```

Comment:
`introduced_in` field is missing (same thing with the other pki commands)

Reply:
done